src/general/base_stage/EditorBase.cs

```diff
@@ -827,6 +828,11 @@ protected virtual void ElapseEditorEntryTime()
         throw new GodotAbstractMethodNotOverriddenException();
     }
 
+    protected virtual void UpdatePatchDetails()
```

Comment:
I think this should have a summary comment explaining when this is triggered and what the purpose is because there's no way to tell from the name alone what this method is about.